site/content/api/Metrics.md

```diff
@@ -40,11 +40,12 @@ a way to transmit metrics to InfluxDB resulting in the same database structure.
 The InfluxDB configuration file used for the example deployment contains templates for
 extracting the following *tags* from metric names transmitted via the Graphite reporter.
 
-| Tag      | Description |
-| -------- | ----------- |
-| *host*   | The name of the host that the service reporting the metric is running on. |
-| *type*   | The type of message that the metric is being processed for (either `telemetry` or `event`) |
-| *tenant* | The name of the tenant that the metric is being reported for. |
+| Tag        | Description |
+| ---------- | ----------- |
+| *host*     | The name of the host that the service reporting the metric is running on. |
+| *type*     | The type of message that the metric is being processed for (either `telemetry` or `event`) |
+| *tenant*   | The name of the tenant that the metric is being reported for. |
+| *protocol* | The protocol of message that the metric is being processed for (example: `http`, `mqtt`) |
```

Comment:
I think you have added this protocol tag only for the new command metrics and not for the old events/telemetry?! Since we plan to change the metrics to a tag and micrometer based approach in the near future I also would not change it. But then we should mark, that this in only used for commands.

Reply:
I have added only for the command metrics. Documentation has also been updated.